minho/김서방찾기.js

```diff
@@ -0,0 +1,7 @@
+function solution(seoul) {
+    var answer = '';
+    for(var i=0;i<seoul.length;i++){
+        if(seoul[i]=='Kim')return '김서방은 '+i+'에 있다';
+    }
```

Comment:
```suggestion
    return `김서방은 ${seoul.indexOf('Kim')}에 있다`;
```
indexOf(문자)라는 메서드를 쓰면 배열 안에서 해당 문자가 배열안에 몇번째 인덱스에 있는지 알 수 있고, 문자열 안에 명령어를 쓰고 싶다면 ``안에 ${명령어}를 쓰시면 됩니다!